commit f3579250f8aa332f2015cb1352ed8b7373eb10db
Author: James Cloos <cloos@jhcloos.com>
Date:   Wed Aug 20 10:32:52 2008 -0400

    xaw8 is gone, use xaw7

diff --git a/configure.ac b/configure.ac
index a219b35..dc33c8a 100644
--- a/configure.ac
+++ b/configure.ac
@@ -61,7 +61,7 @@ if test x$LOCALMANPATH != x; then
 fi
 
 # Checks for pkg-config packages
-XAW_CHECK_XPRINT_SUPPORT(XMAN)
+PKG_CHECK_MODULES(XMAN, xaw7)
 
 AC_ARG_WITH(manconfig,
         AC_HELP_STRING([--with-manconfig=<filename>],
--- xman-1.0.3/Makefile.am~	2008-11-11 17:52:20.675110977 +0100
+++ xman-1.0.3/Makefile.am	2008-11-11 17:53:07.190316386 +0100
@@ -50,12 +50,6 @@
         vendor.h \
         version.h
 
-if XAW_USE_XPRINT
-xman_SOURCES += \
-        print.c \
-        print.h
-endif
-
 appman_PRE = \
         xman.man
 
@@ -63,13 +57,8 @@
 
 appdefaultdir = @appdefaultdir@
 
-if XAW_USE_XPRINT
-Xman: Xman-xprint.ad
-	ln -s $(top_srcdir)/Xman-xprint.ad Xman
-else
 Xman: Xman-noxprint.ad
 	ln -s $(top_srcdir)/Xman-noxprint.ad Xman
-endif
 
 appdefault_DATA = Xman
 
